tomcat/base_test.go

```diff
@@ -382,4 +352,74 @@ func testBase(t *testing.T, context spec.G, it spec.S) {
 		})
 
 	})
+
+	context("$BPI_TOMCAT_ADDITIONAL_JARS is set", func() {
+		it.Before(func() {
+			Expect(os.Setenv("BPI_TOMCAT_ADDITIONAL_JARS", "/layers/test-buildpack/foo/bar.jar")).To(Succeed())
+		})
+
+		it.After(func() {
+			Expect(os.Unsetenv("BPI_TOMCAT_ADDITIONAL_JARS")).To(Succeed())
+		})
+
```

Comment:
```suggestion
	context("$CLASSPATH is set", func() {
		it.Before(func() {
			t.Setenv("CLASSPATH", "/layers/test-buildpack/foo/bar.jar")
		})

```

Reply:
I think the name of the env variable is not clarified yet, but I already changed it to `t.Setenv()` - Thanks, I always forget about that.